exaudfclient/base/exaudflib/exaudflib.h

```diff
@@ -1,32 +1,7 @@
 #ifndef EXAUDFLIB_H
 #define EXAUDFLIB_H
 
-#include <zmq.hpp>
-#include <string>
-#include <sstream>
-#include "script_data_transfer_objects_wrapper.h"
-#include <functional>
-#include <map>
-#include <mutex>
-#include <iostream>
 
-using namespace std;
-
-
-void init_socket_name(const string socket_name);
-bool send_init(zmq::socket_t &socket, const string client_name);
-void send_close(zmq::socket_t &socket, const string &exmsg);
-bool send_run(zmq::socket_t &socket);
-//bool send_return(zmq::socket_t &socket, const char* result);
-void send_undefined_call(zmq::socket_t &socket, const std::string& fn);
-bool send_done(zmq::socket_t &socket);
-void send_finished(zmq::socket_t &socket);
-
-
-
-
-
-extern void* handle;
 void* load_dynamic(const char* name);
 
 #ifdef PROTEGRITY_PLUGIN_CLIENT
```

Comment:
move to new header udf_plugin_interface.h

Reply:
Rename preporcessor constant PROTEGRITY_PLUGIN_CLIENT to UDF_PLUGIN_CLIENT

Reply:
Done